bitsv/network/services/bchsvexplorer.py

```diff
@@ -0,0 +1,105 @@
+import requests
+import json
+from decimal import Decimal
+
+from bitsv.network import currency_to_satoshi
+from bitsv.network.meta import Unspent
+
+# left here as a reminder to normalize get_transaction()
+from bitsv.network.transaction import Transaction, TxInput, TxOutput
+
+DEFAULT_TIMEOUT = 30
+BSV_TO_SAT_MULTIPLIER = 100000000
+
+
+class BchSVExplorerDotComAPI:
+    """
+    Simple bitcoin SV REST API --> uses Legacy address format
+    - get_address_info
+    - get_balance
+    - get_transactions
+    - get_transaction
+    - get_unspent
+    - broadcast_tx
+    """
+    MAIN_ENDPOINT = 'https://bchsvexplorer.com/'
+    MAIN_ADDRESS_API = MAIN_ENDPOINT + 'api/addr/{}'
+    MAIN_BALANCE_API = MAIN_ADDRESS_API + '/balance'
+    MAIN_UNSPENT_API = MAIN_ADDRESS_API + '/utxo'
+    MAIN_TX_PUSH_API = MAIN_ENDPOINT + 'api/tx/send/'
+    MAIN_TX_API = MAIN_ENDPOINT + 'api/tx/{}'
+    MAIN_TX_AMOUNT_API = MAIN_TX_API
+    TX_PUSH_PARAM = 'create_rawtx'
+
+    headers = {
+        'Content-Type': 'application/json',
+        'Accept': 'application/json',
+    }
+
+    @classmethod
+    def get_address_info(cls, address):
+        r = requests.get(cls.MAIN_ADDRESS_API.format(address), timeout=DEFAULT_TIMEOUT)
+        r.raise_for_status()  # pragma: no cover
+        return r.json()
+
+    @classmethod
+    def get_balance(cls, address):
+        r = requests.get(cls.MAIN_BALANCE_API.format(address), timeout=DEFAULT_TIMEOUT)
+        r.raise_for_status()  # pragma: no cover
+        return r.json()
```

Comment:
I think this needs to be normalized to return in Satoshis.

Reply:
Thanks Ill check it out on weekend...

Reply:
Ok! Sounds good. I am able to proceed on my end, will just enable the bsv bits when we get this all working.